elements/rh-code-block/rh-code-block.css

```
@@ -0,0 +1,49 @@
+:host {
+  position: relative;
+  display: block;
+}
+
+#content {
+  display: block;
+  background-color: var(--rh-color-surface-lighter, #f2f2f2);
+  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle-on-light, #c7c7c7);
+  border-block-start-width: var(--rh-code-block-border-block-start-width,
+    var(--rh-border-width-sm, 1px));
+  padding: var(--rh-space-xl, 24px);
+  font-family: var(--rh-font-family-code, RedHatMono, "Red Hat Mono", "Courier New", Courier, monospace);
+  color: var(--rh-color-text-primary-on-light, #151515);
+  max-width: 1000px;
+  max-height: 640px;
+  overflow-y: scroll;
```

```suggestion
  overflow-y: auto;
```

I think we should hide the scrollbar if we don't need it.  Example: https://codepen.io/heyMP/pen/wvQdwYa?editors=1010